.github/scripts/install_nvidia_utils_linux.sh

```diff
@@ -23,19 +23,17 @@ install_nvidia_driver_amzn2() {
     (
         set -x
 
+        # Purge any nvidia driver installed from RHEL repo
+        sudo yum remove -y nvidia-driver-latest-dkms
+
         HAS_NVIDIA_DRIVER=0
         # Check if NVIDIA driver has already been installed
         if [ -x "$(command -v nvidia-smi)" ]; then
             # The driver exists, check its version next
             INSTALLED_DRIVER_VERSION=$(nvidia-smi --query-gpu=driver_version --format=csv,noheader)
 
             if [ "$INSTALLED_DRIVER_VERSION" != "$DRIVER_VERSION" ]; then
-                # TODO
-                # Remove this after torchrec and FBGEMM have both been updated to use
-                # PyTorch NVIDIA installation script instead of using the latest driver
-                # from RHEL repo
-                HAS_NVIDIA_DRIVER=1
-                echo "NVIDIA driver ($INSTALLED_DRIVER_VERSION) has been installed, but we expect to have $DRIVER_VERSION instead. Skipping NVIDIA driver installation for now until torchrec and FBGEMM are updated to use PyTorch NVIDIA installation script instead of RHEL repo"
+                echo "NVIDIA driver ($INSTALLED_DRIVER_VERSION) has been installed, but we expect to have $DRIVER_VERSION instead. Continuing"
```

Comment:
In this case, do we need to uninstall the pre-installed driver instead? Presumably the driver (if any) here was not installed via yum

Reply:
I have tested this with an older (and newer) version of the driver installation script from S3.  As long as the installation mechanism is the same, the script will be able to seamlessly handle different driver version.  I have seen only 2 ways of installing this 1) NVIDIA scripts and 2) yum across all domain libraries.